app/views/pages/page.html.erb

```diff
@@ -1,33 +1,59 @@
-<div class="grid-row grid-gap">
-  <div class="grid-col-12">
-    <% if @page.expired? %>
-      <div class="usa-alert usa-alert--warning">
-        <div class="usa-alert__body">
-          <h4 class="usa-alert__heading">This page has expired</h4>
-          <p class="usa-alert__text">
-            This page expired on <%= @page.expires_at.strftime("%B %d, %Y") %>
-          </p>
-        </div>
-      </div>
+<% if @page.hero.present? && @page.hero.enabled? %>
```

Comment:
[suggestion, nonblocking] it would be nice if there was a `page.hero_enabled?` method that could do this `present? && enabled?` check for us.